sdk/sdk-common/src/users/implementation/client/PositionClientImpl.ts

```diff
@@ -0,0 +1,39 @@
+import { RiskRatio, TokenAmount } from '~sdk-common/common'
```

Comment:
As discussed in the meeting this morning, I think we will have the following pattern for classes:
`IPosition` -> Interface with all data members
`Position` -> Class extending `IPosition` and adding functionality

I think the `PositionClientImpl` and the server side of it won't exist anymore as this class does not need to communicate with a server counter part to retrieve anything, while classes like `User` do need to communicate with the server and will have the Client/Server parts.

Reply:
Again, I know we discussed it this morning and I'm fine with not changing it in this PR and do it in the serialization PR, just pointing it out so we don't forget :)

Reply:
Thanks for adding your expectation clearly here, I'll see if I can do that and it make sense if not I'll skip as suggested :)